### PR TITLE
Added versionRemoved tag to dashboards.config

### PR DESCRIPTION
**Pull request by poornimanayar**
The `dashboard.config` file is not applicable to `v8`. So I have added the tag `versionRemoved: 8.0.0`

### CODE REVIEW

**Comment by sofietoft** [comment]
Check out my comment here: #2163 (comment)

File: Reference/Config/dashboard/index.md [hunk 1]
```diff
@@ -1,128 +1,7 @@
...
-        </areas>
-        <tab caption="Welcome">
-            <control>/app_plugins/mycustomdashboard/customwelcome.html</control>
-        </tab>
-    </section>
-</dashBoard>
-```
-
-[Check out our Creating a Custom Dashboard tutorial](../../../Tutorials/Creating-a-Custom-Dashboard/index.md)
+Custom dashboards can be registered using a package.manifest(../../../Extending/Dashboards/#registering-with-packagemanifest) or using [code](../../../Extending/Dashboards/#registering-with-c-type).
```
Comment (sofietoft) [inline Reference/Config/dashboard/index.md]: Looks like you missed some brackets here? 😄 

```suggestion
Custom dashboards can be registered [using a package.manifest](../../../Extending/Dashboards/#registering-with-packagemanifest) or using [code](../../../Extending/Dashboards/#registering-with-c-type).
```

File: Reference/Config/dashboard/index-v7.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+---
+versionFrom: 7.0.0
```
Comment (sofietoft) [inline Reference/Config/dashboard/index-v7.md]:
```suggestion
versionFrom: 7.0.0
versionRemoved: 8.0.0
```

**Comment by poornimanayar** [comment]
Sorry about missing out on the `versionRemoved` tags

**Comment by poornimanayar** [comment]
And this one too :-)

**Comment by sofietoft** [comment]
No worries at all @poornimanayar ! 😄 

Thanks again for putting in all the work here.
I'll have this merged right away.

**Pull request merged**